com.rustdesk.RustDesk.json

```diff
@@ -67,9 +69,10 @@
           "sha256": "725f8d53b397add3fbab24f793baa086391d1bb1a03ff80ceacc4d96597d2e63",
           "dest-filename": "rustdesk.deb",
           "x-checker-data": {
-            "type": "anitya",
-            "project-id": "309593",
-            "url-template": "https://github.com/rustdesk/rustdesk/releases/download/$version/rustdesk-$version-aarch64.deb"
+            "type": "json",
+            "url": "https://api.github.com/repos/rustdesk/rustdesk/releases/latest",
+            "version-query": ".tag_name",
```

Comment:
Why is "timestamp-query" only available for the x86_64 variant? You probably just missed it?
```suggestion
            "timestamp-query": ".published_at",
            "version-query": ".tag_name",
```

Reply:
As I mentioned above, we could also add it for `aarch64`, but I think it's redundant. The bot will only update the metainfo file once, and so having `timestamp-query` only on the `x86_64` source is enough.